hs_core/templates/pages/baseresource.html

```diff
@@ -15,7 +15,6 @@
     <link rel="stylesheet" href="{% static 'css/Leaflet.fullscreen.min.css' %}" />
     <link rel="stylesheet" href="https://unpkg.com/leaflet-draw@1.0.4/dist/leaflet.draw.css" />
 
-{% if not page|is_debug %}
```

Comment:
There is another page that needs to be updated. https://github.com/hydroshare/hydroshare/blob/develop/theme/templates/pages/homepage.html#L256

Reply:
thanks good call!

Reply:
Now that I look more closely, I see that these 2 entries are likely sufficient to keep our pages from being indexed on dev machines:
https://github.com/hydroshare/hydroshare/blob/4948-robots/theme/templates/base.html#L6
https://github.com/hydroshare/hydroshare/blob/4948-robots/theme/templates/newbase.html#L6

So maybe this PR is unnecessary! :)

Reply:
Beta and prod currently have the following setting:
![image](https://user-images.githubusercontent.com/17934193/217614114-fef31047-598f-45ae-91a9-6bdd5ce21102.png)
So I think it is good to keep the management command in this PR, so that any other locations that currently exist or those that might be added also don't get indexed